app/components/availability_component.html.erb

```diff
@@ -1,4 +1,11 @@
-<span class="text-gray-900 text-sm <%= "font-medium" if @developer.available_now? %>">
-  <%= t(".#{@developer.availability_status}_html", date: date, date_in_words: date_in_words) %>
-</span>
-<%= inline_svg_tag "icons/solid/check_circle.svg", class: "h-5 w-5 text-green-400 mx-2" if @developer.available_now? %>
+<div class="flex items-center text-gray-900">
+  <% if available? %>
+    <%= inline_svg_tag "icons/solid/check_circle.svg", class: "h-5 w-5 text-green-500" %>
+  <% elsif show_unavailable_icon? %>
+    <%= inline_svg_tag "icons/solid/x_circle.svg", class: "h-5 w-5 text-gray-400" %>
+  <% end %>
```

Comment:
Great call. I extracted `AvailabilityComponent#availability_icon`. An `IconComponent` could work, too, but I don't want that until we need to use it somewhere else.

Reply:
I was thinking the `IconComponent` would encapsulate the [default] icon logic, thus it would be called anywhere there is currently an `inline_svg_tag`. These calls are always specifying `class: "h-5 w-5` which could form the basis of the class on the component. This would probably fall outside the scope of this PR.